app/assets/stylesheets/application/controllers/works.scss

```diff
@@ -148,4 +148,18 @@
       margin-top: 20px;
     }
   }
+
+  .info {
+    .new-edit-request-btn {
+      margin: 14px 0 10px;
+    }
+
+    .new-edit-request-btn > i {
```

Comment:
Merge rule `.new-edit-request-btn > i` with rule on line 153<br>Selector should have depth of applicability no greater than 3, but was 4